ci-operator/step-registry/openshift-extended/web-tests/openshift-extended-web-tests-commands.sh

```diff
@@ -8,6 +8,12 @@ pwd && ls -ltr
 cd frontend || exit 0
 pwd && ls -ltr
 
+#set env for kubeadmin
+if [ -f "${SHARED_DIR}/kubeadmin-password" ]; then
+    QE_KUBEADMIN_PASSWORD=$(cat "${SHARED_DIR}/kubeadmin-password")
+    export QE_KUBEADMIN_PASSWORD
```

Comment:
how about just use `KUBEADMIN_PASSWORD`?

Reply:
I could change it, but was just trying to be consistent [openshift-extended-test-commands.sh](https://github.com/openshift/release/blob/master/ci-operator/step-registry/openshift-extended/test/openshift-extended-test-commands.sh#L107)